### PR TITLE
Add function to convert LaneSegment to numpy array. Compute length of 3D polylines.

### PR DESCRIPTION
**Pull request by pd-lars**

### CODE REVIEW

File: paralleldomain/model/geometry/polyline_3d.py [hunk 1]
```diff
@@ -26,6 +26,16 @@ class Line3DBaseGeometry(Generic[T]):
     start: Point3DBaseGeometry[T]
     end: Point3DBaseGeometry[T]
 
+    @property
+    def direction(self) -> Point3DBaseGeometry[T]:
+        """Returns the directional vector of the line."""
+        return self.end - self.start
+
+    @property
+    def magnitude(self) -> float:
```
Comment (phillip-thomas-pd) [inline paralleldomain/model/geometry/polyline_3d.py:35]: also name this length. Otherwise it gets confusing when the length is the sum of magnitudes

Comment (pd-lars) [reply paralleldomain/model/geometry/polyline_3d.py:35]: It was called length before, but I tried to stay consistent with the naming in `Line2DBaseGeometry`. I'm happy to change it in both, though.

Comment (phillip-thomas-pd) [reply paralleldomain/model/geometry/polyline_3d.py:35]: Good point. Lets rename it in Line2DBaseGeometry as well then. Could you also add the length property to Polyline2D and Polygon2D and Polygon3D?

Comment (pd-lars) [reply paralleldomain/model/geometry/polyline_3d.py]: Renamed it in Line2DBaseGeometry Polygon2D / Polygon3D both inherit from their respective Polyline classes, so no need to re implement the length property here.

**Pull request merged**